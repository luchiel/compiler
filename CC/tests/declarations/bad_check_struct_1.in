int main()
{
struct { int b; } *a;
a.b = 5;
}