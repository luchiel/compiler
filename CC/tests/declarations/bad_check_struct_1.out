Error: ParseError:4:2 Left must be struct
